### PR TITLE
Fix database edit

### PR DESCRIPTION
**Pull request by hsolorza**
fixes issue #211 

## Changes in this PR.

- [X] Fix db edit

## Testing this PR.

1. start Iam 
2. view the db edit

### Expected Output.

```
$ rake spec
[...]
```

@osuosl/devs

### CODE REVIEW

File: routes/database.rb [hunk 1]
```diff
@@ -35,6 +35,7 @@ def self.registered(app)
           MyLog.log.fatal 'routes/database: Database Resource not found [edit]'
           halt 404, 'database resource not found'
         end
+        @project = Project.filter(id: @db.project_id).all.first
         @projects = Project.all
```
Comment (Kennric) [inline routes/database.rb]: it looks  like `@projects` being used to populate the project pull-down list.. it feels a bit redundant to query for one project and also query for all of them. I think you can just get the specific project out of `@projects` - that's a Sequel dataset, so you can filter it without doing a new query. So get Projects.all, then get this specific project out of that dataset like `@projects.where(:id => @db.project_id)`

File: views/database/edit.erb [hunk 2]
```diff
@@ -26,7 +25,7 @@
           </td>
         </tr>
         <tr>
-          <td>Project Id</td>
+          <td>Project Name</td>
```
Comment (Kennric) [inline views/database/edit.erb]: You can just say 'Project' here

File: routes/database.rb (hunk 1 above)
Comment (alxngyn) [inline routes/database.rb]: i agree
It should be `@projects = Project.filter(id: @db.project_id).all` (without the first)
then in the view, it should iterate through the projects

File: views/database/edit.erb [hunk 3]
```diff
@@ -26,7 +25,7 @@
...
           <td>
             <% if @db %>
```
Comment (alxngyn) [inline views/database/edit.erb]: If a DB exists without a parent project
this line is where the view is trying to parse a nil object

should change to something like `<% if @project %>`

**Pull request merged**